QuasistaticBrownianThermalNoise/QuasistaticBrownianThermalNoise.cpp

```diff
@@ -454,10 +462,48 @@ double ElasticProblem<dim>::getYijkl(int whichYijkl, unsigned int i,
   return 0.0;
 }
 
+// Switch case to find Yijkl using more than one loss angle for the coating.
+template <int dim>
+double ElasticProblem<dim>::getMultLossAngleYijkl(int whichYijkl,
+                                                  unsigned int i,
+                                                  unsigned int j,
+                                                  unsigned int k,
+                                                  unsigned int l) {
+  switch (whichYijkl) {
```

Comment:
Ok for now, but open an issue for later to change the switches into something more modern, like std::map, where you just give the key and it returns the right thing. Then you get compiler errors, not runtime errors, if you ask for an undefined material. But this isn’t worth it until we get science results for multi loss angles